qiskit/transpiler/passes/routing/sabre_swap.py

```diff
@@ -225,6 +225,12 @@ def run(self, dag):
                 .with_lookahead(0.5, 20, SetScaling.Size)
                 .with_decay(0.001, 5)
             )
+        elif self.heuristic == "depth":
+            heuristic = (
+                Heuristic(attempt_limit=10 * num_dag_qubits)
+                .with_basic(1.0, SetScaling.Size)
+                .with_depth(0.5, SetScaling.Size)
+            )
```

Comment:
If we're exposing this publicly, we need to list the `depth` option in the docstring, and we probably need to make a similar modification to `SabreLayout`.

Reply:
Yes, you're right, I’ve added the depth option to the docstring and corrected a typo there.

`SabreLayout` doesn’t mention the heuristic in its docstring, so I don’t think a similar modification is needed for it at this point. This does remind me, though, that in the future, we might want to add an option for the heuristic in `SabreLayout`. Currently, it defaults to 'decay', which could be limiting when we want to run `SabreLayout` without `SabreSwap`.

Reply:
Hmm, we probably should expose the swap heuristic as an option on `SabreLayout` too. But we can do this in a follow up PR since it's a new API change and probably out of scope for this PR.